src/main/resources/db/changelog/changelog.xml

```diff
@@ -28,27 +28,21 @@
     
     CREATE TABLE AsyncQuery (
        id varchar(36) NOT NULL,
+       asyncAfterSeconds int,
+       requestId varchar(36),
        principalName varchar(255),
        query varchar(1024) NOT NULL,
        queryType int NOT NULL,
        status int NOT NULL,
        createdOn timestamp NOT NULL,
        updatedOn timestamp NOT NULL,
        naturalkey varchar(255) NOT NULL,
-       PRIMARY KEY (id)
-     );
-
-     CREATE TABLE AsyncQueryResult (
-       id varchar(36) NOT NULL,
+       completedOn timestamp NOT NULL,
        contentLength integer NOT NULL,
        responseBody varchar(10485760) NOT NULL,
```

Comment:
should be nullable

Reply:
all the columns that moved from AsyncQueryResult to AsyncQuery should be nullable.